floris/tools/optimization/scipy/yaw.py

```diff
@@ -171,24 +197,84 @@ def _optimize(self):
         Returns:
             opt_yaw_angles (np.array): optimal yaw angles of each turbine.
         """
+        opt_yaw_angles = np.array(self.x0, dtype=float)
+        self._reduce_control_variables()
```

Comment:
based on the discussion in line 243, should these two lines be swapped?

Reply:
This is no longer necessary since self.x0 is no modified within the code.